### DISCUSSION TITLE
Invalid XML feed. Unsure how to fix.

### DISCUSSION THREAD

**Discussion by ItsJustPW** [statamic/starter-kit-podcaster#2] [General]
Not sure if the xml is displaying correctly. It's failing Spotify podcast due to email. When I look at the XML I see this: 

`<itunes:owner>
<itunes:name>XLVets</itunes:name>
<itunes:email/>
</itunes:owner>`

I'm unsure if it's a bug and how to fix it. Any help would be great. Full URL to XML [here](https://podcasts.xlvets.co.uk/feed.xml).

**Comment by jackmcdade** [comment]
It looks like you haven't set the Podcast Owner Email yet. Spotify requires that you do:

<img width="428" alt="CleanShot 2021-10-27 at 07 45 07@2x" src="https://user-images.githubusercontent.com/44739/139059540-ac778747-eb6e-49d5-b03f-d40740230f6f.png">
